homeassistant/components/livisi/const.py

```diff
@@ -7,12 +7,19 @@
 
 CONF_HOST = "host"
 CONF_PASSWORD: Final = "password"
+AVATAR = "Avatar"
+AVATAR_VERSION = "2.0"
 AVATAR_PORT: Final = 9090
 CLASSIC_PORT: Final = 8080
 DEVICE_POLLING_DELAY: Final = 60
 LIVISI_STATE_CHANGE: Final = "livisi_state_change"
 LIVISI_REACHABILITY_CHANGE: Final = "livisi_reachability_change"
 
 SWITCH_PLATFORM: Final = "switch"
+CLIMATE_PLATFORM: Final = "climate"
```

Comment:
I don't think these two constants are any use.
```suggestion
```

Reply:
Done, thank you.